hydroflow/examples/chat/client.rs

```diff
@@ -13,9 +13,19 @@ use tokio_stream::wrappers::LinesStream;
 
 pub(crate) async fn run_client(opts: Opts) {
     // set up network and I/O channels
-    let server_addr: SocketAddr = format!("{}:{}", opts.addr, opts.port).parse().unwrap();
+    let server_ip = match opts.server_addr {
+        Some(ip) => ip,
+        None => panic!("Clients must specify --server-addr"),
+    };
+
+    let server_port = match opts.server_port {
+        Some(port) => port,
+        None => panic!("Clients must specify --server-port"),
+    };
```

Comment:
Rust: simplify this to `let server_port = opts.server_port.expect("Clients must specify --server-port");`